src/database/operations/baseFields/updateBaseField.ts

```diff
@@ -1,21 +1,14 @@
 import { generateCreateOrUpdateItemOperation } from '../generators';
-import type { BaseField, WritableBaseField } from '../../../types';
+import type { BaseField, ShortCode, WritableBaseField } from '../../../types';
 
 const updateBaseField = generateCreateOrUpdateItemOperation<
```

Comment:
This also shouldn't exist!  We can now just use `createOrUpdateBaseField` everywhere.  Life gets so simple.